### ISSUE TITLE
Right click(two-finger click) and drag the mouse have a serious bug

### ISSUE THREAD

**Issue by Etern213** [imbushuo/mac-precision-touchpad#368]
**Pre-report checks**

Yes, but I found nothing to solve this problem.

**Describe the bug**

**two-finger click(Right click) and drag the finger have a serious bug.** When I use one-finger click(Left click) and drag the finger, its work normally. But when I use **two-finger click(Right click) and drag the finger**, its works the same as **one-finger click(Left click) and drag the finger**

**To Reproduce**

Use two-finger click and drag the finger to reproduce the problem. For example, in world of warcraft, Right click and drag the mouse is very commonly used.

**Expected behavior**

I think when I use two-finger click and drag the finger, it should works the same as Right click and drag the mouse.

**Traces**

**Environment**

I use macbook pro(ratina 13-inch early 2015), and install win10 with bootcamp.

[AB#654](https://ligstd.visualstudio.com/7694e0d0-94e3-4fd2-b39a-ecd261e1ba2e/_workitems/edit/654)

**Comment by Etern213** [comment]
Maybe my description is not clear enough, but I think reproduce this problem is very easy. Before I install it, two-finger click and drag is normal, but after I install it, some bug occurred. @imbushuo

**Comment by funtikar** [comment]
you mean it only happens in WoW?

**Comment by Etern213** [comment]
> you mean it only happens in WoW?

no, it happens all the time. I just ... use wow as an example.

**Comment by niklase2** [comment]
I don't think it's a bug, you can't do that with precision drivers in general. I have a Lenovo Windows machine on which i never installed any touchpad drivers myself. The inbuilt trackpad has Windows Precision Drivers. When I click with two fingers and drag (which i never do but I tried it) it does exactly what you described, as if i was only pressing down with one finger. 
I think this is because the two finger click was a feature implemented to help macOS people get used to Windows (double click instead of right click was invented by Apple.) 

TL;DR Although Windows precision drivers have implemented the two finger click, I would advice you to just use the right click instead, because it is the "original" way to do it and I don't think it's a bug that the developper of this can solve (I don't think it's a bug at all but rather an incomplete feature.)

**Comment by TorutheRedFox** [comment]
for me on a MacBookPro9,2, two finger dragging is not possible at all

the mouse just won't budge until you trigger a mouse up, opening the context menu, or whatever releasing a right click is programmed to do in a given window

**Comment by hannad4** [comment]
two finger dragging isn't a thing on windows, that's just a macOS thing....

**Comment by TorutheRedFox** [comment]
sad

it should be a thing as many things use right click dragging

**Comment by Kleysley** [comment]
Press down with two fingers hard enough to engage the mechanical force touch built in to the trackpad. Drag your item(s) to the desired destination and then release the pressure off the button

**Comment by TorutheRedFox** [comment]
there's no such thing as force touch on a mid 2012 macbook pro

**Comment by Kleysley** [comment]
Im sorry I didntmean force touch meant press down with 2 fingers on the touchpad so it produces a physical click. Then let go one finger and drag it with the other. Does that work?

**Comment by TorutheRedFox** [comment]
no that just triggers the button up event for the right mouse button, making the mouse act like I'm not pressing anything
